MyContactList/LMAContactsController.swift

```diff
@@ -10,12 +10,75 @@ import UIKit
 
 class LMAContactsController: UIViewController {
 
+    
+    @IBOutlet weak var em_text: UITextField!
+    @IBOutlet weak var hp_text: UITextField!
+    @IBOutlet weak var cp_text: UITextField!
+    @IBOutlet weak var zp_text: UITextField!
+    @IBOutlet weak var st_text: UITextField!
+    @IBOutlet weak var ct_text: UITextField!
+    @IBOutlet weak var ad_text: UITextField!
+    @IBOutlet weak var ln_text: UITextField!
+    @IBOutlet weak var fn_text: UITextField!
+    
+    @IBOutlet weak var contact_form: UIScrollView!
     @IBOutlet weak var _scrollView: UIScrollView!
+    
+    @IBOutlet weak var ve_switch: UISwitch!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         _scrollView.contentSize = CGSize(width: 320, height: 500)
+        if ve_switch.isOn {
+            em_text.isUserInteractionEnabled = true
```

Comment:
When you notice your code reusing groups of things, set up a collection to group them together once for reuse everywhere else.